### PR TITLE
fix: fix iOS 13+ NSNull source values

### PR DESCRIPTION
**Pull request by ewanharris**
This PR cherry picks the iOS 13 crashing issue out of #103 so that we could safely include an updated module without requiring a semver-major update to the facebook SDK

### CODE REVIEW

**Review by sgtcoolguy — APPROVED** [review]
LGTM

**Comment by lokeshchdhry** [comment]
@ewanharris , @sgtcoolguy , Jenkins is failing to make an IOS module. Can you please check.

**Comment by sgtcoolguy** [comment]
@lokeshchdhry Yeah, I'm desperately trying to make it happy and get a build generated.

File: ios/titanium.xcconfig [hunk 1]
```diff
@@ -4,7 +4,11 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
+<<<<<<< HEAD
 TITANIUM_SDK_VERSION = 7.5.2.GA
+=======
+TITANIUM_SDK_VERSION = 8.1.0.GA
+>>>>>>> aa52d02... build: use sdk 8.1.0.GA to build
```
Comment (hansemannn) [inline ios/titanium.xcconfig]: Merge conflict

Comment (sgtcoolguy) [reply ios/titanium.xcconfig]: @hansemannn You're not supposed to see my mistakes! 😃

**Comment by sgtcoolguy** [comment]
getting closer. Some of the nodes got an updated Xcode 11 gm seed install, so the iOS sims may be taking a while to load on first launch of them leading to the iOS sim timeout right now... I guess I'll just keep retrying

**Comment by vijaysingh-axway** [comment]
@ewanharris @sgtcoolguy Please update the manifest file to make the module version 6.0.1.

**Comment by hansemannn** [comment]
Note that my latest commit in #103 also fixes the UIWebView issues some people have when submitting apps (related to the changes I did for the SDK). Without it, apps may be rejected in the near feature. Time for a SDK 9.0.0 😇

**Comment by sgtcoolguy** [comment]
Closing in favor of #109

**Pull request closed without merging**